lib/index.js

```diff
@@ -44,5 +44,8 @@ History.prototype.lengthForward = require('./prototype/length-forward')
 History.prototype.backward = require('./prototype/backward')
 History.prototype.forward = require('./prototype/forward')
 History.prototype.move = require('./prototype/move')
+History.prototype.forgetAllInDirection = require('./prototype/forget-all-in-direction')
```

Comment:
It seems useful as an internal method, though, just not worth complicating the API that's exposed to library users.